app/services/apply_mods_metadata.rb

```diff
@@ -76,15 +71,9 @@ def log_error!(exception)
     log.puts(exception.backtrace.to_s)
   end
 
-  # Open a new version for the given object if it is in the accessioned state.
+  # Open a new version for the given object if not already open
   def version_object
-    return unless accessioned?
-
-    unless DorObjectWorkflowStatus.new(cocina.externalIdentifier, version: cocina.version).can_open_version?
-      log.puts("argo.bulk_metadata.bulk_log_unable_to_version #{cocina.externalIdentifier}") # totally unexpected
-      return
-    end
-    commit_new_version
+    open_version unless version_service.open?(druid: cocina.externalIdentifier)
```

Comment:
I would just put the code from `open_version` in `version_object`. I'd suggest that it will it clearer what is happening.

Reply:
Fixed, thanks.